plugins/python-build/share/python-build/jython-2.7.0

```diff
@@ -0,0 +1,22 @@
+require_java
+install_jar "jython-2.7.0" "https://repo1.maven.org/maven2/org/python/jython-installer/2.7.0/jython-installer-2.7.0.jar#b44352ece72382268a60e2848741c96609a91d796bb9a9c6ebeff62f0c12c9cf" jython
+
+# Jython is supposed to install pip using ensurepip by default, but this does
+# not appear to be working so we do it manually.
+#
+# pyenv's `ensurepip` downloads `get-pip.py` which isnt currently compatible
+# with jython: http://bugs.jython.org/issue2302
+#
+# Use the ensurepip module in jython to install the bundled pip instead:
+"$PYTHON_BIN" -m ensurepip
```

Comment:
I though that this can be replaced by pyenv's `build_package_ensurepip` since it'll try to run `"$PYTHON_BIN" -m ensurepip` first. It wasn't working as I expected, though.

I'm still not sure why we need to call `$PYTHON_BIN` explicitly, but I confirmed this is working :+1:

https://github.com/yyuu/pyenv/blob/50d5de473168ba159b9035ee592a937c68461bf3/plugins/python-build/bin/python-build#L1532